src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
-body {
+@import 'var';
+
+* {
   margin: 0;
```

Comment:
```suggestion

```
If you want to remove the margin for the body/ul/etc., consider applying this rule directly for needed tags